code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -219,6 +219,7 @@
 	if(cameranet.checkTurfVis(remote_eye.loc))
 		if(X.monkeys >= 1)
 			var/mob/living/carbon/human/monkey/food = new /mob/living/carbon/human/monkey(remote_eye.loc)
```

Comment:
you need a check here to prevent the monkey from being created, at all, if the limit has been reached--and some bluspacey message about it too.